localization/english/activities/expeditions/expedition_activity_l_english.yml

```diff
@@ -62,23 +62,23 @@
 
-  expedition_proving_fields_desc:0 "#D The Proving Fields#! - Ancient stone barracks are all that remains of this once vast compound. Ages ago the headquarters of a famous marital order, now just overgrown ruins. The greatest warriors of the world would gather here to sharpen their skills and watch the true elites of the world spar. 
+  expedition_proving_fields_desc:0 "#D The Proving Fields#! - Ancient stone barracks are all that remains of this once vast compound. Ages ago it was the headquarters of a famous marital order, but has long since been left to the whims of nature. The greatest warriors of the world would gather here to sharpen their skills and watch the true elites of the world spar. 
 
-  expedition_fortress_tomb_of_the_dead_emperor_desc:0 "#D Fortress Tomb Of The The Last Emperor#! - The final resting place of the final elf emperor of earth's history. He retreated to this fortress of solitude after suffering betrayal and ultimate defeat by the hands of a last alliance of elves and men in the Great Rebellion."
+  expedition_fortress_tomb_of_the_dead_emperor_desc:0 "#D Fortress Tomb Of The The Last Emperor#! - The final resting place of the last Elf Emperor that resided on Earth. He retreated to this fortress of solitude after suffering betrayal and absolute defeat at the hands of the Great Rebellion."
 
-  expedition_volcanic_forge_desc:0 "#D Volcanic Forge Of Celebrim#! - Here deep in the volcano is rumored to lie the forge and workshop of Celebrim the greatest crafter the elven peoples of Earth have ever know. An emperor would gladly pay a kingdom for a chance to own one of Celebrim's wondrous creations and wars have been fought for the power that some of them hold."
+  expedition_volcanic_forge_desc:0 "#D Volcanic Forge Of Celebrim#! - Here, deep within the volcano, is rumored to lie the forge and workshop of Celebrim; The greatest artisan the Elves of Earth have ever seen. An emperor would gladly forfeit a kingdom for a mere chance to own one of Celebrim's wondrous creations, and wars have been fought for the power that some of them hold."
 
-  expedition_noble_wood_desc:0 "#D The Noble Wood#! - A forest of trees taller and thicker than castle towers. It was once the realm of a truly powerful elf queen not dark, but beautiful and terrible as the Morning and the Night! Fair as the Sea and the Sun and the Snow upon the Mountain! Dreadful as the Storm and the Lightning! Stronger than the foundations of the earth. It is said all loved her and despaired!”
+  expedition_noble_wood_desc:0 "#D The Noble Wood#! - A forest of trees taller and thicker than most castle spires. It was once the realm of a truly powerful Elf queen; As beautiful and terrible as the morning and the night! As graceful as the sea, and as fair as the snow upon the Alps! As dreadful as the storm and the lightning! Stronger than the foundations of the Earth! It is said that all that knew her loved and despaired.”
```

Comment:
SO this one is a direct reference/quote from Galadriel and must not be altered except for punctuation lol
https://reprog.wordpress.com/2012/12/29/i-wish-jackson-hadnt-ruined-galadriels-speech/

Reply:
My bad, not familiar with that speech lol

Reply:
haha, its a nod for the LOTR fanatics